website/assets/scss/_blog.scss

```diff
@@ -0,0 +1,194 @@
+.blog{
+
+  &-rounded {
+    border-radius: .5 * $blog-gutter-xs;
+  }
+
+  &__pagination {
+    margin: $blog-gutter-md;
+
+    &>li>a {
```

Comment:
We should avoid using element tags like `li` or `a` for styling for performance reasons - because of how css matches rules to elements its faster when we give class names to elements we want to match. So here we can use something like `&__pagination-link`